source/MaterialXCore/Look.cpp

```diff
@@ -136,4 +158,121 @@ vector<VariantAssignPtr> MaterialAssign::getActiveVariantAssigns() const
     return activeAssigns;
 }
 
+//
+// Lookgroup methods
+//
+
+LookVec LookGroup::getActiveLooks() const
+{
+    string looks = getActiveLook();
+    if (looks.empty())
+    {
+        looks = getLooks();
+    }
+    const StringVec& lookList = splitString(looks, ARRAY_VALID_SEPARATORS);
+    LookVec activeLooks;
+    if (!lookList.empty())
+    {
+        vector<LookPtr> lookElements = getDocument()->getLooks();
```

Comment:
Should this also be const vector<LookPtr>& ?

Reply:
No. This is a non-const copy.